core/primitives/src/time.rs

```diff
@@ -82,74 +74,107 @@ impl Drop for MockClockGuard {
     }
 }
 
+pub struct Clock {}
```

Comment:
please comment that the 'Clock' is the class that you're supposed to pass to your 'production' code - and use its methods to fetch time.